backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -134,18 +135,16 @@ public Optional<User> getUserById(String id) {
 		return userRepository.findById(id).map(userMapper::toUser);
 	}
 
-	public void updateUser(String userId, String email) {
+	public void updateUser(String userId, UserPatch userPatched) {
 		Assert.hasText(userId, "userId is required; it must not be null or blank");
-		Assert.hasText(email, "email is required; it must not be null or blank");
+		Assert.notNull(userPatched, "patchedUser is required; it must not be null");
 
 		log.debug("Fetching user [{}] from repository", userId);
 		final var user = userRepository.findById(userId).orElseThrow();
-
-		if (email.equals(user.getEmail()) == false) {
-			user.setEmail(email);
+		if (userPatched.getEmail().equals(user.getEmail()) == false) {
```

Comment:
Since either of these values can be null, you should use `StringUtils.equals(..)` from `org.apache.commons.lang3` to compare.

Reply:
fixed